habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -568,6 +560,7 @@ def _set_ao_states_from_ep(self, ep_info: RearrangeEpisode) -> None:
             ao.joint_positions = ao_pose
 
     def is_point_within_bounds(self, pos):
+        # NOTE: This check is loose: really we want the island bounds, not the full navmesh
```

Comment:
@ASzot surfacing this. Because it is not restricted to the largest indoor island, this check is very loose. Maybe unnecessary, but I didn't want to remove in case it serves a purpose.